python/desc/imsim/ImageSimulator.py

```diff
@@ -34,7 +34,7 @@ class ImageSimulator:
     Class to manage the parallel simulation of sensors using the
     multiprocessing module.
     """
-    def __init__(self, instcat, psf, numRows=None, config=None, seed=267,
+    def __init__(self, instcat, create_centroid_file, psf, numRows=None, config=None, seed=267,
```

Comment:
I would prefer to make `create_centroid_file` a keyword argument with a default value of `False` rather than make it a required argument, so that existing scripts won't break.  I would also add it towards the end of the argument list, say after `apply_sensor_model`.